src/commands/userinfo/userinfoCommand.ts

```diff
@@ -0,0 +1,23 @@
+import { UserinfoInterface } from "../../interfaces/userinfo";
+
+export default class UserinfoCommand {
```

Comment:
same here, let's just create an interface and pass an object around, like authorizationCommand